packages/@aws-cdk/pipelines/test/helpers/compliance.ts

```diff
@@ -0,0 +1,101 @@
+interface Tests {
+  readonly ignore: Tests;
+
+  legacy(fn?: () => void): void;
+  modern(fn?: () => void): void;
+}
+
+interface Test {
+  run(): void;
```

Comment:
Should we give it another name, like `register`? (Since it's not actually running any test at this point).